jsonate/fields.py

```diff
@@ -1,3 +1,4 @@
+from builtins import str
```

Comment:
actually it's simpler than all that (which has mistakes anyway): in python 2 unicode strings should work